<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaViewer</class>
 <widget class="QWidget" name="MediaViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1001</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#centralWidget {
	background-color: rgb(85, 87, 83);
}

QStackedWidget#toolbar {
	min-height: 60px;
}

QFrame#frame {
	border: 0px solid transparent;
	max-height: 65px;
	min-height: 65px;
}

QFrame#frame_2 {
	border: 0px solid transparent;
	max-height: 60px;
	min-height: 60px;
}

QPushButton#button_display {
	max-width: 50px;
	min-width: 50px;
	max-height: 50px;
	min-height: 50px;
	border-image: url(:/res/icons/ic_display.png);
}

QPushButton#button_backward {
	max-width: 40px;
	min-width: 40px;
	max-height: 40px;
	min-height: 40px;
	border-image: url(:/res/icons/ic_video_backward.png);
}

QPushButton#button_forward {
	max-width: 40px;
	min-width: 40px;
	max-height: 40px;
	min-height: 40px;
	border-image: url(:/res/icons/ic_video_forward.png);
}

QPushButton#button_back {
	max-width: 50px;
	min-width: 50px;
	max-height: 50px;
	min-height: 50px;
	border-image: url(:/res/icons/ic_back.png);
}

QPushButton#button_delete {
	max-width: 50px;
	min-width: 50px;
	max-height: 50px;
	min-height: 50px;
	border-image: url(:/res/icons/ic_delete.png);
}

QSlider {
	max-height: 50px;
	min-height: 50px;
	background: rgba(165, 192,120, 0);
}

QSlider::groove {                                 
	border: 1px solid rgba(114,159,207,0);                                 
	height: 4px;  
	border-radius: 3px;                                
	background: rgba(238,238,2,20);                                 
}  
                                                                   
QSlider::add-page  {                                  
	background: qlineargradient(x1:0,y1:0,x2:1,y2:1,stop:0 #8c8c8c, stop:1 #8c8c8c);                                     
	border: 1 px solid rgba(32, 92, 160,0);      
	border-radius: 3px;                               
	width: 5px;                                 
} 
                                                                  
QSlider::sub-page {                                     
	background-color: #1578ba;
	border: 1 px solid rgba(32, 92,160, 0);                
	border-radius: 3px;                            
	width: 5px;                                 
} 
                                                                  
QSlider::handle {                                 
	width: 50px;
	height: 50px;                        
	border-image: url(:/res/icons/ic_display_handle.png);                          
	margin: -23px -13px -23px -13px;                          
}

QLabel#label_display {
	color: rgb(238, 238, 236);
}

QLabel#label_left {
	color: rgb(85, 87, 83);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="centralWidget" native="true">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="1,0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>710</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="zoombar" native="true">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	max-width: 50;
	max-height: 50;
	min-width: 50;
	min-height:50
}

QWidget#zoombar {
	background-color: rgba(255,255,255,0);
	border-style: outset;
	border-width: 0px;
	border-radius: 10px;
	border-color: transparent
}

QPushButton#zoominButton {
	border-image: url(:/res/icons/ic_zoomin.png) 0 0 0 0
}

QPushButton#zoomoutButton {
 border-image: url(:/res/icons/ic_zoomout.png) 0 0 0 0
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,1,1,1,2">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="zoominButton">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="zoomoutButton">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QStackedWidget" name="toolbar">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	max-width: 50;
	max-height: 50;
	min-width: 50;
	min-height:50
}

QWidget#toolbar {
	background-color: rgba(255,255,255,0);
	border-style: outset;
	border-width: 0px;
	border-radius: 10px;
	border-color: transparent

}

QPushButton#gobackButton {
	border-image: url(:/res/icons/ic_back.png) 0 0 0 0
}



QPushButton#deleteButton {
	border-image:url(:/res/icons/ic_delete.png) 0 0 0 0
}

QPushButton#previousButton {
	border-image:url(:/res/icons/ic_backward.png) 0 0 0 0 
}

QPushButton#nextButton {
   border-image:url(:/res/icons/ic_forward.png) 0 0 0 0 
}</string>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="toolbarimage">
         <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,1,0,20,0,1,1">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="1" column="5">
           <widget class="QPushButton" name="gobackButton">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>62</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <spacer name="spacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>22</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="3">
           <spacer name="spacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>124</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="6">
           <spacer name="spacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>22</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="deleteButton">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="toolbarvideo">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,10,1">
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="1" column="0">
              <widget class="QLabel" name="label_display">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <spacer name="horizontalSpacer_8">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>795</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_left">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="3">
              <widget class="QSlider" name="videoSlider">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="20,0,80,0,25,0,25,0,80,0,20">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="button_delete">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>214</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="button_backward">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="button_display">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="button_forward">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>214</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="button_back">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
